yarn-project/noir-contracts/src/libs/noir-aztec/src/constants_gen.nr

```diff
@@ -0,0 +1,90 @@
+// GENERATED FILE - DO NOT EDIT, RUN yarn remake-constants in circuits.js
+global ARGS_LENGTH: comptime Field = 16;
+global RETURN_VALUES_LENGTH: comptime Field = 4;
+global READ_REQUESTS_LENGTH: comptime Field = 4;
+global MAX_NEW_COMMITMENTS_PER_CALL: comptime Field = 4;
```

Comment:
Not sure about some of these GENERATOR__ names but looks good to me

Reply:
I felt like prefixing it with enum name is the best solution given that there are no native enums in Noir. Kev told me that enums can be approximated with structs but that seems like unnecessary complexity to me.

Or would you prefer some other prefix?